www/docs/en/dev/plugin_ref/spec.md

```diff
@@ -17,700 +17,489 @@ license: >
+  ```
+
+  Examples of using these Windows specific attributes:
+  ```  
+  <framework src="src/windows/example.dll" arch="x64" />
+  <framework src="src/windows/example.dll" versions=">=8.0" />
+  <framework src="src/windows/example.vcxproj" type="projectReference" target="win" />
+  <framework src="src/windows/example.vcxproj" type="projectReference" target="all" versions="8.1" arch="x86" />
+  <framework src="src/windows/example.dll" target-dir="bin/x64" arch="x64" custom="true"/>
+  ```
 
-        <asset src="www/new-foo.js" target="js/experimental/foo.js" />
+### info
+
+  Additional information provided to users. This is useful when you require extra steps that can't be easily automated or are beyond plugman's scope.  
 
-  creates the `js/experimental` directory within the `www` directory,
-  unless already present, then copies the `new-foo.js` file and
-  renames it `foo.js`.  If a file already exists at the target
-  location, plugman stops and reverses the installation process,
-  issues a notification about the conflict, and exits with a non-zero
-  code.
+  Example:
+  ```
+  <info>
+  You need to install __Google Play Services__ from the `Android Extras` section using the Android SDK manager (run `android`).
 
-## _js-module_ Element
+  You need to add the following line to the `local.properties`:
 
-Most plugins include one or more JavaScript files.  Each `<js-module>`
-tag corresponds to a JavaScript file, and prevents the plugin's users
-from having to add a `<script>` tag for each file.  While `<asset>`
-tags simply copy a file from the plugin subdirectory into `www`,
-`<js-module>` tags are much more sophisticated. They look like this:
+  android.library.reference.1=PATH_TO_ANDROID_SDK/sdk/extras/google/google_play_services/libproject/google-play-services_lib
+  </info>
+  ```
 
-    <js-module src="socket.js" name="Socket">
-        <clobbers target="chrome.socket" />
-    </js-module>
+### hook
 
-When installing a plugin with the example above, `socket.js` is copied
-to `www/plugins/my.plugin.id/socket.js`, and added as an entry to
-`www/cordova_plugins.js`. At load time, code in `cordova.js` uses XHR
-to read each file and inject a `<script>` tag into HTML. It adds a
-mapping to clobber or merge as appropriate, as described below.
+  Represents your custom script which will be called by Cordova when certain action occurs (for example, after plugin is added or platform prepare logic is invoked). This is useful when you need to extend default Cordova functionality. See [Hooks Guide](../guide/appdev/hooks/index.html) for more information.
 
-_Do not_ wrap the file with `cordova.define`, as it is added
-automatically. The module is wrapped in a closure, with `module`,
-`exports`, and `require` in scope, as is normal for AMD modules.
+  Example: 
+  ```
+  <hook type="after_plugin_install" src="scripts/afterPluginInstall.js" />
+  ```
 
-Details for the `<js-module>` tag:
+### uses-permission and variables
```

Comment:
The header for this seems strange. It can probably just be uses-permission. Also, I think we should make it very clear in the first sentence what platforms support this. All of the examples are for Android (and `uses-permission` is also the name of a commonly used tag in `AndroidManifest.xml`), but iOS is mentioned in the last paragraph?